docs/DeveloperGuide.md

```diff
@@ -283,16 +283,155 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `ContaX` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: List persons**
+
+**MSS**
+
+1. User requests to list persons
+2. ContaX shows a list of persons
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The list is empty
+
+  Use case ends. 
```

Comment:
This portion can be removed, because the extension does not change the fact that the use case ends (as opposed to shorting the use case for subsequent cases).

Reply:
Fixed.